### PR TITLE
Feat/precompiled search

### PR DESCRIPTION
**Pull request by KonnorRogers**
Precompiles search via Babel and allows you to seperately import it by appending `/dist`. Should not affect any users currently using the older  version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,9 +20,20 @@
   "files": [
     "frontend"
   ],
+  "scripts": {
+    "prepare": "npm run build",
```
Comment (jaredcwhite) [inline package.json]: not a big deal, but I'd like to stick with the convention of using Yarn rather than NPM

Comment (jaredcwhite) [reply package.json]: Actually these just run the build script…any reason we need `prepare` and `prepack`?

**Review by jaredcwhite — APPROVED** [review]
looks good, just one question about the scripts

File: package.json (hunk 1 above)
Comment (KonnorRogers) [reply package.json]: These are purely for me running from my personal repo. They can be removed. They're just a lifecycle hook for git repos! Sorry!

**Comment by jaredcwhite** [comment]
@ParamagicDev hmm, not sure why but I had to put `import "bridgetown-quick-search/frontend/dist"` for it to work…just /dist would choke when Webpack tried to bundle.

**Comment by KonnorRogers** [comment]
Weird. Javascript gives me headaches.

**Pull request merged**